src/components/event-listing/EventListing.tsx

```diff
@@ -60,7 +62,7 @@ const EventListing: React.FC<EventProps> = ({
           <Text strong>{dateFormat(details.start, 'shortTime', true)}</Text>
           <ThinDivider />
           <Paragraph ellipsis={{ rows: 5 }}>{details.description}</Paragraph>
-          <GreenButton>Register</GreenButton>
+          <GreenButton to={`events/${id}`}>Register</GreenButton>
```

Comment:
should it be `/events/${id}`? I think if there's no leading slash, it will append the url to the current URL. I might have that backwards.

Can you type LinkButton in frontend-scaffold? That's also causing some warnings.